### PR TITLE
Fill in the 0.0.7 changelog

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,8 +27,41 @@
...
+- [Object reform][gtk221] benefits include:
+
+  - Automatic generation of [the majority of code][auto-dir], expanding the
+    number of implemented classes, eliminating typos and making maintenance
+    easier.
+
+  - Unified upcasting and downcasting via the `Cast` trait, proper interface
+    support. In particular this makes `Builder` safe to use.
+
+  - The [`prelude`][prelude] module, which reexports all traits from `gtk` and
```
Comment (gkoz) [inline CHANGELOG.md]: This bit is slightly untrue right now but I'll open a PR to actually reexport all the dependencies' traits.

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,8 +27,41 @@
...
+- New [safe event bindings][gdk95] are [passed to signal handlers][gtk244].
+  These bindings don't have many getters and setters implemented yet but the low
+  level event structs are still reachable [via `as_ref`][examples68].
+
+- New safe [`Error`][glib86-error] and [`Value`][glib95] bindings. `ListStore` and
+  `TreeStore` [changed accordingly][gtk238], traded `set_string` for more
+  powerful [`set`][gtk241].
+
+- In an ongoing move to [reflect integer unsignedness properly][gtk246]
+  `Notebook` and partly `ListStore` and `TreeStore` switched from `i32` to
```
Comment (GuillaumeGomez) [inline CHANGELOG.md]: This sentence seems a bit weird. What about "Notebook and partly both ListStore and TreeStore"? (yes, this __both__ changes everything :p )

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,8 +27,41 @@
...
+
+  - Automatic generation of [the majority of code][auto-dir], expanding the
+    number of implemented classes, eliminating typos and making maintenance
+    easier.
+
+  - Unified upcasting and downcasting via the `Cast` trait, proper interface
+    support. In particular this makes `Builder` safe to use.
+
+  - The [`prelude`][prelude] module, which reexports all traits from `gtk` and
+    its dependencies and some ubiquitoue types (`Continue`, `Inhibit`).
```
Comment (GuillaumeGomez) [inline CHANGELOG.md]: "ubiquitoue" -> "ubiquitous"

File: CHANGELOG.md (hunk 1 above)
Comment (GuillaumeGomez) [inline CHANGELOG.md]: What do you prefer? Waiting for the PR to be opened or to edit it afterwards? I'm fine with both solutions.

**Comment by GuillaumeGomez** [comment]
Excellent work! Please fix the typos and we're good to go! :)

File: CHANGELOG.md (hunk 1 above)
Comment (gkoz) [inline CHANGELOG.md]: I'd rather merged this now.

**Comment by gkoz** [comment]
Rephrased the signedness bullet.

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,8 +27,41 @@
...
+  These bindings don't have many getters and setters implemented yet but the low
+  level event structs are still reachable [via `as_ref`][examples68].
+
+- New safe [`Error`][glib86-error] and [`Value`][glib95] bindings. `ListStore` and
+  `TreeStore` [changed accordingly][gtk238], traded `set_string` for more
+  powerful [`set`][gtk241].
+
+- In an ongoing move to [reflect integer unsignedness properly][gtk246]
+  `Notebook` switched from `i32` to `u32`, `ListStore` and `TreeStore` partly
+  switched to `u32`, `EntryBuffer` switched to `u16`.
```
Comment (GuillaumeGomez) [inline CHANGELOG.md]: "and EntryBuffer switched to"

Comment (gkoz) [inline CHANGELOG.md]: This isn't a mistake though is it? :)

Comment (GuillaumeGomez) [inline CHANGELOG.md]: No it's not, but it's better. :)

**Comment by gkoz** [comment]
Updated.

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**